cmd/spec/path.yaml

```diff
@@ -107,6 +107,21 @@ paths:
       operationId: getPubkeyList
       tags:
         - Pubkey
+      parameters:
+        - in: query
+          name: limit
+          schema:
+            type: integer
+            default: 100
+          required: false
+          description: The number of items to return
+        - in: query
+          name: offset
+          schema:
+            type: integer
+            default: 0
+          required: false
+          description: The number of items to skip before starting to collect the result set.
```

Comment:
Can you research if there is a way to create a `component` that can be referenced in parameters? That would save us a lot of copy-pasting.